content/en/docs/security/contextual-authorization.md

```diff
@@ -0,0 +1,63 @@
+
+For example, by granting a given Virtual Machine (or principal that such machine
+operates under) access to a AWS S3, applications running inside that machine can
+request a token on-demand, which would enable them access to the AWS S3 buckets
+without having to store long lived credentials anywhere.
+
+By leveraging such capability, Flux users can focus on the big picture, which is
+access controls enforcment with a least privileged approach, whilst not having to
+do deal with security hygine topics such as encrypting authentication secrets, and
+ensure they are being rotated regularly.
+All that is taken care automatically by the cloud providers, as the tokens provided
+are context and time bound.
+
+## Current Support
+
+Below is a list of Flux features that support this functionality and their documentation:
+
+| Status | Component | Feature | Provider | Ref |
+| ------ | ----------------- | ------- | -------- | ------ |
+| Supported | Source Controller | Bucket Repository Authentication | AWS | [Guide][AWS Buckets] |
+| Supported | Source Controller | Bucket Repository Authentication | Azure | [Guide][Azure Buckets] |
+| Supported | Source Controller | Bucket Repository Authentication | GCP | [Guide][GCP Buckets] |
+| Supported | Source Controller | OCI Repository Authentication | AWS | [Guide][AWS OCI Repository]  |
+| Supported | Source Controller | OCI Repository Authentication | Azure | [Guide][Azure OCI Repository]  |
+| Supported | Source Controller | OCI Repository Authentication | GCP | [Guide][GCP OCI Repository]  |
+| Supported | Image Reflector Controller | Container Registry Authentication | AWS | [Guide][AWS Container Registry] |
+| Supported | Image Reflector Controller | Container Registry Authentication | Azure | [Guide][Azure Container Registry] |
+| Supported | Image Reflector Controller | Container Registry Authentication | GCP | [Guide][GCP Container Registry] |
+| Supported | Kustomize Controller | SOPS Integration with Cloud KMS | AWS | [Guide][AWS KMS] |
+| Supported | Kustomize Controller | SOPS Integration with Cloud KMS | Azure | [Guide][Azure KMS] |
+| Supported | Kustomize Controller | SOPS Integration with Cloud KMS | GCP | [Guide][GCP KMS] |
+
+## Roadmap
+
+Support for context-based authorization should only increase over time.
+
+For more information, please visit the tracking issue: https://github.com/fluxcd/flux2/issues/3003.
+
+
+[AWS Buckets]: https://fluxcd.io/docs/components/source/buckets/#aws
+[Azure Buckets]: https://fluxcd.io/docs/components/source/buckets/#azure
+[GCP Buckets]: https://fluxcd.io/docs/components/source/buckets/#gcp
+[AWS OCI Repository]: https://fluxcd.io/docs/components/source/ocirepositories/#aws
+[Azure OCI Repository]: https://fluxcd.io/docs/components/source/ocirepositories/#azure
+[GCP OCI Repository]: https://fluxcd.io/docs/components/source/ocirepositories/#gcp
+[AWS Container Registry]: https://fluxcd.io/docs/guides/image-update/#using-native-aws-ecr-auto-login
```

Comment:
Might be better relative links for the rest?  e.g `../guides/image-update.md#using-native-aws-ecr-auto-login`.
I think this looks great. Thanks @pjbgf !

Reply:
Thank you, made all links relative.